docs/source/API/core/initialize_finalize/is_Initialized.rst

```diff
@@ -0,0 +1,56 @@
+``is_initialized``
+==================
+
+.. role::cpp(code)
+    :language: cpp
+
+Defined in header ``<Kokkos_Core.hpp>``
+
+Usage: 
+
+.. code-block:: cpp
+
+    Kokkos::is_initialized();
+
+
+Queries the initialization status of Kokkos and returns ``true`` if Kokkos is initialized and ``false`` is Kokkos is not initialized. This function can be called prior or after Kokkos initialization or finalization.
+
+Interface
+---------
+
+.. code-block:: cpp
+
+    Kokkos::is_initialized();
+    
+Requirements
+~~~~~~~~~~~~
+
+* ``Kokkos::is_initialized`` can be called before ``Kokkos::initialize``.
+* ``Kokkos::is_initialized`` can be called after ``Kokkos::finalize``.
+
+Semantics
+~~~~~~~~~
+
+* After calling ``Kokkos::initialize``, ``Kokkos::is_initialized()`` should return true.
+
+Example
+~~~~~~~
+
+.. code-block:: cpp
+
+    #include <Kokkos_Core.hpp>
+    #include <cassert>
+
+    int main(int argc, char* argv[]) {
+        assert(!Kokkos::is_initialized());
+        Kokkos::initialize(argc, argv);
+	assert(Kokkos::is_initialized());
+        Kokkos::finalize();
+        assert(!Kokkos::is_initialized());
```

Comment:
Hmm should we mention as a note in this part of the doc, that once Kokkos is finalized it is not allowed to re-initialize?